vscode/package.json

```diff
@@ -447,7 +447,28 @@
       "[qsharp]": {
         "debug.saveBeforeStart": "none"
       }
-    }
+    },
+    "extensionDependencies": [
+      "github.copilot-chat"
+    ],
+    "chatParticipants": [
+      {
+        "id": "quantum.copilot",
+        "name": "quantum",
```

Comment:
fullName should be provided too (see <https://code.visualstudio.com/api/extension-guides/chat>)

Reply:
Added fullname: "Azure Quantum Copliot"

Reply:
Per the docs for fullName on the link, "...which is shown in the title area of a response from your participant". So it should be relatively brief, and "Copilot" is probably unnecessary and redundant if it's in the copilot chat window.

I suggest just `Azure Quantum` (maybe even just `Quantum` would be fine, but marketing would probably like us to have the `Azure` in there).

Reply:
Okay, I've changed it to "Azure Quantum".